### DISCUSSION TITLE
更改文件夹重命名规则

### DISCUSSION THREAD

**Discussion by 1525745393** [jxxghp/MoviePilot#2246] [问题咨询]
大佬怎么把文件夹名改成：庆余年.2023.S01E01
视频文件名改成：庆余年2023.S01.E01

**Comment by cng10001** [comment]
一般就照着readme上写支持的配置项吧，你这种完全个人需求的，还找别人问怎么改，你还是自己根据jinjia2语法看怎么配置吧- -
